add usage for expression filter (#753)
pingcap/docs-dm: en/overview.md

In the MySQL ecosystem, tools such as gh-ost and pt-osc are widely used. DM provides support for these tools to avoid migrating unnecessary intermediate data. For details, see [Online DDL Tools](key-features.md#online-ddl-tools)

### Filter certain row changes using SQL expressions

> In the phase of incremental replication, DM supports the configuration of SQL expressions to filter out certain row changes, which makes you more specifically control the data to be replicated. For more detailed information, refer to [Filter Certain Row Changes Using SQL Expressions](feature-expression-filter.md).

Reviewer: ```suggestion
In the phase of incremental replication, DM supports the configuration of SQL expressions to filter out certain row changes, which lets you replicate the data with a greater granularity. For more information, refer to [Filter Certain Row Changes Using SQL Expressions](feature-expression-filter.md).
```
